automation_tools/__init__.py

```diff
@@ -486,13 +486,29 @@ def setup_foreman_discovery():
 def setup_libvirt_key():
     """Task to setup key pairs for secure comminication between Satellite server
```

Comment:
Please also documented expected env variables. For example see `vm_create()` function.